### ISSUE TITLE
TG2 parte 3

### ISSUE THREAD

**Issue by Jpobletel** [Exploratorio-DCC-PUC/Syllabus#463]
Tenemos una duda con la primera pregunta de la parte 3, ya que una vez que intentamos entrenar el algoritmo nos aparece este error. No sabemos si podemos preguntar este tipo de cosas pero llevamos como 4 horas pegados sin saber que hacer buscando en google. (Los datos ya estan normalizados del 0 al 1 y por lo menos en nuestro caso elegimos 4 columnas como se ve en el codigo)

![Screenshot_1](https://user-images.githubusercontent.com/69923022/98844922-c6a7a000-242b-11eb-8dc6-67e61ed8542f.png)

**Comment by felipegarcia98** [comment]
Perfect. Pareciera que el problema es que el label que le estas dando es del tipo continua, lo que significa que son decimales, lo cual sería [útil para una regresión](https://scikit-learn.org/stable/modules/neighbors.html) (eso quizás te ayude para el bonus). Para los clasificadores se requiere un label discreto, lo cual involucra números, categorías (str), otros.

No encuentro que sea mayor problema este código, ya que es esencialmente lo mismo que se ve en la ayudantía. Quizás podría ser tema lo de las columnas, pero imposible saber si dropeaste o cambiaste de orden antes

PD: Sobre el tema puedes ver en [StackOverflow](https://stackoverflow.com/questions/41925157/logisticregression-unknown-label-type-continuous-using-sklearn-in-python). Cuando busques errores, te recomiendo pegar el mensaje de error y no reducirte necesariamente al algoritmo en específico, a veces las soluciones vienen de otros lados (incluso otras áreas)
